### ISSUE TITLE
WAV File Playback Only Plays Once

### ISSUE THREAD

**Issue by baykovr** [baykovr/AVPI#54]
Reported that, 32bit .wav files only trigger and play once for Play_Sound action types.

Expected behavior: sound files are played on every invocation.
Actual behavior: the sound file plays only on the first invocation.

**Comment by Tim0theus** [comment]
I forked your branch and added a RandomPicker. As I noticed, that a file only played once I added:
```
                wavePlayer.Pause();
                audioFileReader.Seek(0, System.IO.SeekOrigin.Begin);
```
in the run() of the Play_Sound action class, before the init and it worked for me.

**Comment by baykovr** [comment]
Fixed via 6d93259445e06b691448e2fd0baccc43eb9acf1d
